src/js/angular/core/directives/yasgui-component/yasgui-component.directive.js

```diff
@@ -320,6 +337,8 @@ function yasguiComponentDirective(
 
                     $scope.ontotextYasguiConfig = config;
 
+                    setInitialQueryState();
```

Comment:
Would be better if the `setInitialQueryState` returns a promise and when it is resolved then the `afterInit` to be called.

Reply:
yeah sure, I thought about it, but left it as is. Will think about it again and eventually change it.

Reply:
fixed